multisite/models.py

```diff
@@ -170,9 +170,9 @@ class Alias(models.Model):
     site = models.ForeignKey(
         Site, related_name='aliases', on_delete=models.CASCADE
     )
-    is_canonical = models.NullBooleanField(
+    is_canonical = models.BooleanField(
         _('is canonical?'),
-        default=None, editable=False,
+        default=False, editable=False,
```

Comment:
@kutera, I suggest removing `default=False` since the default value of BooleanField is None when `default=` isn’t defined. Then you don't need the change in `get_queryset`.